governance.md

```diff
@@ -68,26 +66,29 @@ The foundations of Project governance are:
 -   Active Contribution
 -   Institutional Neutrality
 
-Traditionally, Project leadership was provided by a BDFL (Fernando Perez) and
-subset of Contributors, called Core Developers, whose active and consistent
-contributions have been recognized by their receiving “commit rights” to the
-Project GitHub repositories. In general all Project decisions are made through
-consensus among the Core Developers with input from the Community. The BDFL
-can, but rarely chooses to, override the Core Developers and make a final
-decision on a matter.
-
-While this approach has served us well, as the Project grows and faces more
-legal and financial decisions and interacts with other institutions, we see a
-need for a more formal governance model. Moving forward The Project leadership
-will consist of a BDFL and Steering Council. We view this governance model as
-the formalization of what we are already doing, rather than a change in
-direction.
+Project leadership was provided by the original author, John
+D. Hunter.  Shortly before his passing in 2012, leadership was
+transferred to Michael Droettboom, who later invited Thomas Caswell as
+a co-lead.  Additional leadership has also been provided by a subset
+of Contributors, called Core Developers, whose active and consistent
+contributions have been recognized by their receiving “commit rights”
```

Comment:
this does not make sense... from my reading it seems like you mean to say that an individual receiving "commit rights" indicates acknowledgement of their active and consistent contributions.

As the text stands it should read...

>[their] contributions have been recognized by **someone**, which has led to them receiving commit rights to...

but that still sounds clunky...

>Core Developers provide additional leadership and have "commit rights" to the Project repositories; Core Developers comprise of a subset of Contributors whose active and consistent contributions have been recognised by **someone**.

I still feel a little confused here by the language... by ``Core Leadership`` do you refer to those listed on https://github.com/orgs/matplotlib/teams as belonging to the ``Core`` group, or to us ``Developers`` as well.  From the description it sounds like you refer to all of us in the ``Developers`` team, but then you don't list @efiring and @jenshnielsen as part of the Project Leadership.  We should ensure consistency between the names we use here and the names of the teams in the project.